src/Sulu/Bundle/CategoryBundle/Category/KeywordManagerInterface.php

```diff
@@ -32,7 +32,7 @@ interface KeywordManagerInterface
      *
      * @return KeywordInterface
      */
-    public function save(KeywordInterface $keyword, CategoryInterface $category, $force = null);
+    public function save(KeywordInterface $keyword, CategoryInterface $category, $force = null, bool $isPersisted = false);
```

Comment:
@wachterjohannes remove isPersisted again (we use `isNew` from the entity directly)